libs/agno/agno/storage/session/agent.py

```diff
@@ -32,6 +32,13 @@ class AgentSession:
     # Agent Data: agent_id, name and model
     agent_data: Optional[Dict[str, Any]] = None
 
+    # List of all messages in the session
+    chat_history: Optional[list[Dict[str, Any]]] = None
+    # List of all runs in the session
+    runs: Optional[list[Dict[str, Any]]] = None
```

Comment:
This should ideally be a `list[RunResponse]`. We can serialize when we store it.